.doc_gen/metadata/acmpca_metadata.yaml

```diff
@@ -169,4 +154,4 @@ acm-pca_CreatePrivateCertificateAuthorityAD:
               snippet_tags:
                 - acmpca.CreatePrivateCertificateAuthorityAD.main
   services:
-     acm-pca: {}
\ No newline at end of file
+     acm-pca: {CreateCertificateAuthority}
```

Comment:
similarly, not clear to me what this does, if could explain it when the build link working that'd be great